### ISSUE TITLE
Workload clusters not coming up because of Docker registry rate limit / set registry credentials for workload clusters

### ISSUE THREAD

**Issue by AndiDog** [giantswarm/roadmap#2357]
This is particularly problematic while a cluster is bootstrapped, since if kubelet fails to initialize (during cloud-init), no more nodes will be able to join.

- https://github.com/giantswarm/roadmap/issues/2141#issuecomment-1505015005
- https://github.com/giantswarm/roadmap/issues/1912 – But not sure if the mirror is used in case of HTTP 4xx errors. Even if used, using a mirror only hides the problem of missing Docker registry credentials.
- https://github.com/giantswarm/roadmap/issues/1978 also only hides the problem.

**Comment by AndiDog** [comment]
As discussed: for MCs, we could create a separate secret only for `connectivity.containerRegistries` including credentials. Then it could be referenced from the cluster `App` of MC and even its WCs. It would still mean that the app platform renders the credentials into the manifest, though.

**Comment by fiunchinho** [comment]
I don't think this is relevant anymore after migration to the new registry. Shall we close it @AndiDog ?

**Comment by AndiDog** [comment]
Let's close. For auto-heal, the health checks would be more important.
